elements/pf-label-group/demo/label-group-add-label-custom-form.html

```diff
@@ -0,0 +1,300 @@
+<pf-label-group id="label-group" closeable>
+  <h2 slot="category">group name</h2>
+  <pf-label color="red" icon="info-circle" removable>label-1</pf-label>
+  <pf-label color="green" icon="info-circle" removable>label-2</pf-label>
+  <pf-label color="blue" icon="info-circle" removable>label-3</pf-label>
+  <pf-label id="add-button" class="add-button" color="blue" variant="outline" dismissible="false" >
+    Add label </pf-label>
+</pf-label-group>
+
+<script type="module">
+  import '@patternfly/elements/pf-label-group/pf-label-group.js';
+  import '@patternfly/elements/pf-label/pf-label.js';
+  import '@patternfly/elements/pf-button/pf-button.js';
+  import '@patternfly/elements/pf-modal/pf-modal.js';
+  import '@patternfly/elements/pf-select/pf-select.js';
+  import '@patternfly/elements/pf-icon/pf-icon.js';
+
+  const group = document.getElementById('label-group');
+  const addButton = document.getElementById('add-button');
+  console.log('addButton', addButton);
+  
+  
+
+  // create a modal element with form controls and buttons, return it
+  function createModal() {
+    const modal = document.createElement('pf-modal');
+    modal.id = 'custom-label-modal';
+    modal.setAttribute('variant', 'small');
+    modal.setAttribute('aria-describedby', 'custom-label-modal-description');
+    console.log('modal is created');
+
+    modal.innerHTML = `
+      <h2 slot="header">Add label</h2>
```

Comment:
This modal content is static, so put it in the HTML rather than in javascript, and call showModal() on it when you need to